hw/ip/pwm/doc/dv/index.md

```diff
@@ -138,21 +129,21 @@ The functional coverage plan can be found here: [coverageplan](#testplan)
 
 ### Self-checking strategy
 #### Scoreboard
-The `pwm_scoreboard` is primarily used for end to end checking.
+The `pwm_scoreboard` is primarily used for transaction-by-transaction checking.
 It creates the following analysis ports to retrieve the data monitored by corresponding interface agents:
 * item_fifo[NUM_PWM_CHANNELS]: the FIFO w.r.t channels receives the dut items sent by the pwm_monitor
-* exp_item_q[PWM_NUM_CHANNELS]: the queues  w.r.t channels are used to store the expected/referenced items 
+* exp_item                   : It is a generated item used to store the expected/referenced items
```

Comment:
This made more sense when talking about a queue, now the sentence makes little sense. Perhaps you meant just say "It is used to store the expected item constructed from tl address and data channels."?

Reply:
I have updated the file with your review comment